compiler/il/OMRNode.hpp

```diff
@@ -668,6 +668,8 @@ class OMR_EXTENSIBLE Node
    TR::Node *             getVirtualCallNodeForGuard();
 
    TR_OpaqueMethodBlock * getOwningMethod();
+   static TR_OpaqueMethodBlock* getOwningMethod(TR::Node *node, TR::Compilation *comp, bool isIProfilingEnabled);
+   static TR_OpaqueMethodBlock* getOwningMethod(TR_ByteCodeInfo &bcInfo, TR::Compilation *comp);
```

Comment:
Both of these new methods will need some doxygen. The isIProfilingEnabled flag does not make sense to me and probably doesn't make sense in OMR in general. Why do we need this flag and why does it always cause the implementation to return null when the flag is set to true? Something doesn't seem right here...

Reply:
Could you change this to have the comp object first - this is general convention in the compiler.